### ISSUE TITLE
Return the shortcut instance from make_shortcut

### ISSUE THREAD

**Issue by bollwyvl** [newville/pyshortcuts#2]
Adjacent to #1, I've been doing some post-processing of shortcuts. It would be nice if the convenient `make_shortcut` returned the final written location (or the `Shortcut` instance) such that it could be part of more complex pipelines.

Thanks again!

**Comment by newville** [comment]
@bollwyvl   yes, good point. I agree.  Returning the Shortcut instance is probably best.
